Error while checking build/O2/fullCI for 11e37055ff1ac7f8fbc9de98fe433bb58dd4c90b at 2022-10-01 14:58:
```
## sw/BUILD/O2Physics-latest/log
/sw/slc8_x86-64/O2/9966-slc8_x86-64-local1/include/SimulationDataFormat/MCTrack.h:24:10: fatal error: TMCProcess.h: No such file or directory
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/9966/11e37055ff1ac7f8fbc9de98fe433bb58dd4c90b/build_O2_fullCI/pretty.html).

--- a/Detectors/HMPID/calibration/src/HMPIDDCSProcessor.cxx
+++ b/Detectors/HMPID/calibration/src/HMPIDDCSProcessor.cxx
@@ -743,7 +743,6 @@ void HMPIDDCSProcessor::finalize()
   std::unique_ptr<TF1> pPhotMean;
   pPhotMean.reset(new TF1("HMP_PhotEmean", Form("%f", eMean), 0,
                           mTimeEMean.last - mTimeEMean.first));
-  //  std::vector<TF1>::iterator itArNmean = arNmean.begin() + 42;
 
   pPhotMean->SetTitle(Form("HMP_PhotEmean; Time [mS]; Photon Energy [eV]"));
 

